src/pages/settings/Security/CloseAccountPage.js

```diff
@@ -110,9 +116,11 @@ class CloseAccountPage extends Component {
                         text={this.props.translate('closeAccountPage.closeAccount')}
                         isLoading={this.props.closeAccount.isLoading}
                         onPress={() => User.closeAccount(this.state.reasonForLeaving)}
-                        isDisabled={Str.removeSMSDomain(userEmailOrPhone).toLowerCase() !== this.state.phoneOrEmail.toLowerCase()}
+                        isDisabled={Str.removeSMSDomain(userEmailOrPhone).toLowerCase() !== this.state.phoneOrEmail.toLowerCase() || this.props.network.isOffline}
                         style={[styles.mt5]}
                     />
+                    {!this.props.isSmallScreenWidth
+                        && <OfflineIndicator containerStyles={[styles.mt2]} />}
```

Comment:
What exactly do you mean by that Alex? Just want to make sure I understand as a non-engineer 😄

Reply:
Yeah good question, so my train of thought is:
- If we're adding the Offline Indicator to the "Close Account" page, why wasn't it there before?
- Then I checked if the Offline Indicator is present on other form pages (example: the Profile Page) and I don't see the offline indicator there
- It seems weird to me to show the user they're offline in the Close Account page w/out also showing they're offline in the Profile page, so I'm basically asking here if we want to show the Offline Indicator on **all** pages, or only on pages with forms, or something else

One other idea I had is: In web / desktop full screen mode, it may not be necessary to show the Offline Indicator twice (you can see that it's already showing in the chat report page) so maybe we **only** show the Offline Indicator on sub-pages on small screens (including mobile devices)?

Reply:
> It seems weird to me to show the user they're offline in the Close Account page w/out also showing they're offline in the Profile page, so I'm basically asking here if we want to show the Offline Indicator on all pages, or only on pages with forms, or something else

Maybe I misunderstood. But aren't these different offline patterns? We let users modify their profiles offline, but we shouldn't let users delete accounts while offline.

Reply:
Hmm true true, but doesn't it feel a bit weird seeing an offline status in some pages and not in all?

Reply:
I don't think so. By that logic we would be forced to make all settings offline OR non-offline. I don't think we should care about a consistent offline behavior on a feature that lets users stop being our customer

Reply:
IMO, the message `You appear to be offline` is good to know everywhere or nowhere, not "at the bottom of some page but not all".

> I don't think we should care about a consistent offline behavior **on a feature that lets users stop being our customer**

I'm a bit confused how this would let users stop being our customer, do you mean if users see `You appear to be offline` they'll assume the form they're seeing won't be submittable, so they'll think our app doesn't work offline even though some pages will and some won't?

Reply:
OH. Whoops.

I was mistaken and thought this was regarding disabling the Close Account button while offline... not the indicator. Ignore me

Reply:
Aahhhh ok that makes more sense @Julesssss 😅 

> May be worth a discussion in channel, why it was only implemented for mobile and not web and then we can implement it.

Ooh that's a good idea 👍

Reply:
Shall we leave that discussion for now? I think we could merge this today and revisit.

> why it was only implemented for mobile and not web and then we can implement it.

As for this, I think it works now because the page is shown as a RightDockedModal, right?